cluster-autoscaler/cloudprovider/aws/api/converter.go

```diff
@@ -0,0 +1,52 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package api
+
+import "strconv"
+
+// stringRefToFloat64 converts fields of type *float64 to float64 (fallback to zero value if nil)
```

Comment:
string to float?

Reply:
Yes, it's used when transforming AWS API DTO's to local ones. 
For instance at this place: https://github.com/kubernetes/autoscaler/pull/486/files#diff-fc9a082c5fda90baeac6f105e7a018b9R61

Would you suggest to not put it in a function?

Reply:
To be more specific, I believe this is the change wanted:

`of type *float64 to float64` => `of type *string to float64`